### PR TITLE
add PrivilegedProjectMemberProvider

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: add new provider: PrivilegedProjectMemberProvider to support admin access for all projects

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/routing.go [hunk 1]
```diff
@@ -117,17 +119,19 @@ func NewRouting(
...
-		eventRecorderProvider:                 eventRecorderProvider,
-		exposeStrategy:                        exposeStrategy,
-		accessibleAddons:                      accessibleAddons,
-		userInfoGetter:                        userInfoGetter,
-		settingsProvider:                      settingsProvider,
-		adminProvider:                         adminProvider,
-		admissionPluginProvider:               admissionPluginProvider,
-		settingsWatcher:                       settingsWatcher,
+		privilegedProjectMemberProvider:       privilegedProjectMemberProvider,
+
```
Comment (maciaszczykm) [inline api/pkg/handler/routing.go]: Remove the empty line.

Comment (zreigz) [reply api/pkg/handler/routing.go]: done

**Pull request merged**